### PR TITLE
#11815 Fix a typo in the t.w.pages.errorPage docstring

### LINKED ISSUE #11815: twisted.web.pages.errorPage docstring has a typo
> Use [IResource.putChild()](https://docs.twisted.org/en/stable/api/twisted.web.resource.IResource.html#putChild) override specific path.

Should be:

> Use [IResource.putChild()](https://docs.twisted.org/en/stable/api/twisted.web.resource.IResource.html#putChild) override **a** specific path.

### PR DESCRIPTION
**Pull request by twm**
## Scope and purpose

Fixes #11815

## Contributor Checklist:

This process applies to *all* pull requests - no matter how small.
Have a look at [our developer documentation](https://docs.twisted.org/en/latest/core/development/dev-process.html) before submitting your Pull Request.

Below is a non-exhaustive list (as a reminder):

* The title of the PR should describe the changes and starts with the associated issue number, like “#1234 Brief description”.
* A release notes news fragment file was create in src/twisted/newsfragments/ (see: [Release notes fragments docs.](https://docs.twisted.org/en/latest/core/development/dev-process.html#release-notes-management))
* The automated tests were updated.
* Once all checks are green, request a review by leaving a comment that contains exactly the string `please review`.
  Our bot will trigger the review process, by applying the pending review label
  and requesting a review from the Twisted dev team.

### CODE REVIEW

**Comment by twm** [comment]
please review

File: src/twisted/web/pages.py [hunk 1]
```diff
@@ -83,7 +83,7 @@ def errorPage(code: int, brief: str, detail: str) -> IResource:
     useful for rendering simple error pages.
 
     The resource dynamically handles all paths below it. Use
-    L{IResource.putChild()} override specific path.
+    L{IResource.putChild()} override a specific path.
```
Comment (exarkun) [inline src/twisted/web/pages.py]:
```suggestion
    L{IResource.putChild()} to override a specific path.
```

**Review by adiroiban — APPROVED** [review]
Thanks Tom for the change and Jean-Paul for the suggestion.

Tom, please review and apply the suggestion from Jean-Paul and after that this can be merged.

Cheers

**Comment by twm** [comment]
Thank you @exarkun and @adiroiban!

**Pull request merged**